checkout-service/client/src/components/QuantityDropDown.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const QtyButton = styled.div`
+margin-top: 10px;
+height: 20px;
+width: 61px;
+border: 1px solid black;
+line-height: 20px;
+border-radius: 8px;
+border-color: #D5D9D9;
+background: #F0F2F2;
+border-style: solid;
+border-width: 1px;
+box-shadow: 0 2px 5px rgb(15 17 17 / 15%);
+cursor: pointer;
+&:hover {
+  background: #e7e9ec;
+}`;
+
+const QtyButtonText = styled.span`
+font-family: "Amazon Ember",Arial,sans-serif;
+font-size: 11.5px;
+padding-left: 6px;
+&:hover {
+
+}`;
+
+const QtyDropDownWrapper = styled.div`
+width: 61px;
+margin: 0 auto;
+position: absolute;
+cursor: pointer;
+z-index: 1000;
```

Comment:
Qty drop down is the inline element. When z-index was not used, did you run into any issue with elements display ? And, just curious, how did you decide on this number ?

Reply:
So this was the first dropdown I made. I can't recall if I had any issues prior to using z-index. As for the number, I used a random number, I could've used any other besides one but I chose 1000.

Reply:
Got it. Thanks!